.github/workflows/config.yml

```diff
@@ -4,6 +4,24 @@ on: [push]
 
 jobs:
 
+  snyk_scan:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.14'
+      - uses: snyk/actions/setup@master
+      - uses: actions/checkout@v2
+      - name: Snyk Test
+        run: snyk test  --severity-threshold=medium --org=cloudability
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+      - name: Snyk monitor
+        run: snyk monitor --org=cloudability
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
```

Comment:
Just to double check, this only runs on the master build right?

Reply:
No, it runs every commit. I think that's what circleci does currently?

Reply:
Yeah, looks like you are right. Strange that. I think this means we will register all branches to be monitored going forward, even if they've since been merged in.

Reply:
I could update to have the test step run on every commit and the monitor just run on the master build? Though maybe we want to maintain legacy behavior from circleci for now.